File: 26 Remove Duplicates from Sorted Array/removeDuplicatesFromSortedArray.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        k = 0;
        
        if len(nums) == 0: # If the array is empty
            return 0;
        else: # If it is not empty
            for i in range(1,len(nums)):
                if nums[i] > nums[i-1]:
                    if i != k:
                        k += 1
                        nums[k] = nums[i]
                    else:
                        k += 1
            return k+1

File: 26 Remove Duplicates from Sorted Array/removeDuplicatesFromSortedArray.java
public class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 0;
        
        if(nums.length == 0){ // If the array is empty
            return 0;
        }else{ // If it is not empty
            for(int i = 1; i<nums.length; i++){
                if(nums[i] > nums[i-1]){
                    if(i != k)
                        nums[++k] = nums[i];
                    else
                        k++;
                }       
            }
            return k+1;
        }
    }
}